core/src/main/java/hudson/console/AnnotatedLargeText.java

```diff
@@ -215,14 +221,13 @@ protected boolean delegateToWriteLogTo(StaplerRequest2 req, StaplerResponse2 rsp
         return isHtml(req);
     }
 
-    /**
-     * Strips annotations using a {@link PlainTextConsoleOutputStream}.
-     * {@inheritDoc}
-     */
     @CheckReturnValue
     @Override
     public long writeLogTo(long start, OutputStream out) throws IOException {
-        return super.writeLogTo(start, new PlainTextConsoleOutputStream(out));
+        if (isHtml())
+            return writeHtmlTo(start, new OutputStreamWriter(out, StandardCharsets.UTF_8));
```

Comment:
This seems like a potentially incompatible change. Is there a need for it?

Reply:
Indeed, this is not needed. I've built the chain of PRs locally and HTML rendering works in the pipeline-graph-view-plugin. Removed in 2f981dcd02ac7a259f0b5f43c5b19d5bb9764543.